packages/algob/src/builtin-tasks/deploy.ts

```diff
@@ -35,10 +35,10 @@ export function loadFilenames (directory: string, taskType?: string): string[] {
     }
   }
 
-  const jsFiles = glob.sync(path.join(directory, "*.js")).sort(cmpStr);
-  if (jsFiles.length) { return jsFiles; }
-  // if not js file, return the compiled ts files from ./build
-  return glob.sync(path.join("build", directory, "*.js")).sort(cmpStr);
+  // return .js, .ts files from /scripts directory
```

Comment:
Please add this comment as doc string of function

Reply:
ok